Process_Library/MieQuery.H

```diff
@@ -87,76 +87,73 @@
     endif
 #endif
 
-    rh_ = reshape(min(max(rh,0.),0.99),[size(rh)])    ! no (super) saturation 
-    qrh = max(1,nint((rh_+0.001)*1000.))              ! quantized RH
-    arh = this%rha(qrh)                               ! interpolation weight in table
-    irh = this%rhi(qrh)                               ! index in table for each RH vcalue
+    rh_ = min(max(rh,0.),0.99)                                  ! no (super) saturation 
+    qrh = reshape(max(1,nint((rh_+0.001)*1000.)), [size(rh_)])  ! quantized RH
+    arh = reshape(this%rha(qrh), shape(rh_))                    ! interpolation weight in table
+    irh = reshape(this%rhi(qrh), shape(rh_))                    ! index in table for each RH vcalue
```

Comment:
The point of this approach was supposed to be that we could skip the `reshape()` steps.

Reply:
It is hard to avoid. If we put it inside subroutine interp, it will have to be calculated every time